### ISSUE TITLE
Unable to drag a table row to the top of the list when reordering an admintable

### ISSUE THREAD

**Issue by mcjackson18** [craftcms/cms#10781]
### Description
We have a client who has hundreds of discounts, when they need to reorder the discounts they're unable to drag the discount to the top of the list.

For example in the image below, to drag discount 'Test 8' to the top of the list, you need to drag it to above 'Test 3' let go, scroll the list and drag again. When there's hundreds of discounts this isn't practical, you should be able to drag the discount to the top of the list in one go.
 
![discount-reorder](https://user-images.githubusercontent.com/39083512/156211182-0f3ac9ce-d8f5-4df5-bc72-cfd8891b9898.png)

### Additional info

- Craft CMS version: 3.7.34
- Craft Commerce version: 3.4.11

**Comment by brandonkelly** [comment]
3.7.42 is out now with the fix for this.
